doc/panes/editor.rst

```diff
@@ -38,6 +38,14 @@ You can separate cells by lines starting with either:
 * ``# <codecell>`` (IPython notebook cell separator)
 
 Providing a description to the right of the separator will give that cell its own name in the :guilabel:`Outline Explorer`.
+You can also create "subsections" by adding ``%`` signs to the cell separator, e.g. ``# %%%`` to create a level 2 subsection, ``# %%%%`` for level 3, etc...
+This creates multiple outline levels in the the :guilabel:`Outline Explorer`:
```

Comment:
```suggestion
This displays multiple levels in the the :guilabel:`Outline Explorer`.
```

* Avoid word repetition ("create", "outline")
* Swap colon for period (per our convention for introducing figures)